cost-analyzer/values.yaml

```diff
@@ -57,10 +58,14 @@ global:
           window: weekly # or 7d
           aggregation: namespace
           filter: kubecost
-          ownerContact: # ownerContact(s) should be the same for the same namespace, otherwise the last namespace alert overwrites
```

Comment:
has this changed?

Reply:
it has not; every alert will look to send to its individually scoped recipients before defaulting to globalAlertEmails

Reply:
oh I just removed the example from the ones after the first one to avoid repeating it too much. That behavior will be noted in the documentation.

Reply:
all good, you can remove! thanks for clarifying